app/views/notes/edit.html.erb

```diff
@@ -1,3 +1,3 @@
 <%= turbo_frame_tag @note do %>
   <%= render 'form', note: @note %>
-<% end %>
\ No newline at end of file
+<% end %>
```

Comment:
The last line in the file should be the empty one but without whitespace.
Otherwise, you see this icon in the PR 
![Screenshot 2023-05-04 at 5 09 45 PM](https://user-images.githubusercontent.com/56089936/236330899-e6553764-0f60-49f4-83a6-cbc35940800c.png)